sdks/java/core/src/test/java/org/apache/beam/sdk/transforms/CombineFnsTest.java

```diff
@@ -298,9 +298,8 @@ public String apply(String input) {
     assertThat(displayData, hasDisplayItem("combineFn1", combineFn1.getClass()));
     assertThat(displayData, hasDisplayItem("combineFn2", combineFn2.getClass()));
 
-    String nsBase = DisplayDataCombineFn.class.getName();
-    assertThat(displayData, includesDisplayDataFrom(combineFn1, nsBase + "#1"));
-    assertThat(displayData, includesDisplayDataFrom(combineFn2, nsBase + "#2"));
+    assertThat(displayData, includesDisplayDataFor("combineFn1", combineFn1));
```

Comment:
We should create a JIRA and a TODO for switching the composed combine fn to use paths rather than namespace for the duplicates (note we're not doing it yet since rendering doesn't yet support paths).